src/custom-elements/src/components/date/date.tsx

```diff
@@ -279,6 +283,7 @@ export class Date {
 
     this.value = currentValues.filter(value => value !== formattedDate).join(',');
     this.eventChange.emit(this.value);
+    this._bubbledEventChange.emit(this.value);
```

Comment:
Emitting _bubbledEventChange from <chi-date> should be avoided. It emits chiDateChange event that needs to be retargeted.

Reply:
Ack! I think i've found a better solution that doesn't require this extra emitter. Thanks!